goldenmaster/Plugins/TbEnum/Source/TbEnum/Private/Generated/OLink/TbEnumEnumInterfaceOLinkSource.h

```diff
@@ -0,0 +1,68 @@
+/**
+Copyright 2021 ApiGear UG
+Copyright 2021 Epic Games, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#pragma once
+
+#include "TbEnumEnumInterfaceInterface.h"
+#include "Subsystems/GameInstanceSubsystem.h"
+THIRD_PARTY_INCLUDES_START
+#include "olink/iobjectsource.h"
+THIRD_PARTY_INCLUDES_END
+#include "UnrealOLinkHost.h"
+
+class TbEnumEnumInterfaceOLinkSource : public ApiGear::ObjectLink::IObjectSource
```

Comment:
done

Reply:
MINOR - just my opinion, from the description I wouldn't have the whole concept of what is the purpose of this class and how can I use it. The info I miss the most is probably "a helper class for UTbEnumEnumInterfaceOLinkAdapter" (so in general user don't have to care about it, I know it is suggested by having it as a private, but I think it would be worth to mention) maybe also add an info that it wraps/adapts/exposes the service? I know it has the source in name of the class, but for someone not familiar with olink won't be meaningful.